docs/tutorials/get-started/blink-an-led.md

```diff
@@ -0,0 +1,416 @@
+tags: ["board", "raspberry pi", "app", "sdk"]
+authors: []
+languages: [ "python", "go" ]
+viamresources: [ "board" ]
+level: "Beginner"
+date: "4 October 2022"
+cost: 70
+no_list: true
+---
+
+This tutorial will show you how to use Viam to make an LED blink with a Raspberry Pi.
+This is a great place to start if you have never built a robot or a circuit before.
+
+{{<gif webm_src="/tutorials/img/blink-an-led/image9.webm" mp4_src="/tutorials/img/blink-an-led/image9.mp4" alt="A GIF of the completed project showing a blinking blue LED connected to a Raspberry Pi with jumper cables." max-width="300px">}}
+
+First, you'll use the control interface on the [Viam app](https://app.viam.com) to turn the LED on and off.
+Then, you'll write code to control the LED using the Viam [software development kits](../../../program/).
+
+## What you'll need for this guide
+
+### Hardware
+
+- [Raspberry Pi 3 or 4](https://a.co/d/5Tn67G3)
+  - Refer to the [Viam Raspberry Pi Setup Guide](/installation/prepare/rpi-setup/) to set up your Pi.
+- [A solderless breadboard](https://www.amazon.com/dp/B0135IQ0ZC)
+- [Jumper wires](https://amzn.to/2qVhd4y)
+- [Resistor pack](https://amzn.to/2Dmainw)
+  - You will use a 100 Ohm resistor, which is the resistor with brown-black-brown bands
+- [An LED](https://amzn.to/2Ex2v5q)
+
+### Software
+
+- [viam-server](/installation/#install-viam-server)
+- [Go](https://go.dev/dl/) or [Python 3.9+](https://www.python.org/downloads/)
+- [Viam Python SDK](https://python.viam.dev/) or [Viam Go SDK](https://pkg.go.dev/go.viam.com/rdk/robot/client#section-readme)
+
+## Project setup
+
+Before you build your circuit, you need to set up the operating system on your Raspberry Pi and install `viam-server` on the Pi:
+
+1. Follow the [Raspberry Pi Setup Guide](/installation/prepare/rpi-setup/) to set up the operating system.
+2. Then [install `viam-server` and connect your Pi to the Viam app](/installation/#install-viam-server).
+
+{{< snippet "social.md" >}}
+
+## Build the circuit
+
+The first step in this project is to build a simple LED circuit consisting of an LED and a resistor.
+The resistor protects the LED by limiting the flow of electricity through the circuit and is called a *current-limiting resistor*.
+
+Then you will make the LED circuit controllable from the Raspberry Pi by connecting the circuit to the [pins](https://en.wikipedia.org/wiki/Pin_(electronics)) on the Raspberry Pi.
+
+### Understand GPIO pinout
+
+A *general-purpose input/output* (GPIO) pin is a digital signal pin on a circuit board, like a Raspberry Pi, which may be used as an input or output, or both, and is controllable by software.
+
+<img src="../../img/blink-an-led/pi-gpio.png" alt="Photo showing a Raspberry Pi 4 with a white box around the GPIO pins on the Pi and big red letters that say, 'GPIO Pins.'" width="100%">
+
+As you may have guessed, **each pin has a specific role, and you can use it only for that role**.
```

Comment:
```suggestion
**Each pin has a specific role, and you can use it only for that role**.
```
The as you may have guessed isn't needed - we don't want to make them feel stupid if they didn't think that

Reply:
100%